spec/request/api/v1/confirmations/show_confirmation_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe 'GET /api/v1/auth/confirmation', type: :request do
```

Comment:
I would also test that the confirmation email is sent, but on another file as it is part of the registration process, with something like:
```
expect { post registration_path, params: params, as: :json }
  .to change { ActionMailer::Base.deliveries.count }.by(1)
```